scripts/monorepo_vars.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+export CORE_SUBPATH="$(git rev-parse --show-toplevel)/packages/sourcecred"
```

Comment:
Let's derive the core path programmatically using the sh and lerna:
```suggestion
CORE_PACKAGE_SUBPATH=$(lerna ll --loglevel=silent | grep ^sourcecred' ' | cut --delimiter=' ' --fields=3)
export CORE_PATH="$(git rev-parse --show-toplevel)/$CORE_PACKAGE_SUBPATH"
```